docs/specs/verification.md

```diff
@@ -0,0 +1,66 @@
+# User Verification
+
+## Definitions
+
+A user is verified if the user fulfill the condition specified by criteria.
+
+An identity is verifiable if:
+- it is an SSO identity; or
```

Comment:
Curious, why SSO identity = verifiable?

Reply:
We assumes SSO providers (e.g. Google, Facebook, etc.) are trusted: they would ensure the email returned is verified. Therefore, All SSO identities are considered verified.

Since it is more intuitive that only 'verifiable' identities can be 'verified' identities, SSO identities are considered 'verifiable' and always 'verified'.

Reply:
Noted, agreed on the assumption.